srcts/src/components/errorConsole.ts

```diff
@@ -200,6 +200,42 @@ class ShinyErrorConsole extends LitElement {
     });
   }
 
+  static renderClientMessage({ headline, message }: ShinyClientMessage) {
```

Comment:
I find the word `render` a bit misleading here

```suggestion
  static createClientMessageElement({ headline, message }: ShinyClientMessage) {
```